examples/vercel-ai-chatbot/app/actions.ts

```diff
@@ -118,12 +118,7 @@ export async function shareChat(id: string) {
     }
   }
 
-  const payload = {
-    ...chat,
-    sharePath: `/share/${chat.id}`
```

Comment:
Another set of incremental changes I want to make is just simplify the template to reduce the amount of moving pieces